stripe/src/main/java/com/stripe/android/view/CardInputWidget.java

```diff
@@ -743,6 +777,18 @@ protected void onLayout(boolean changed, int l, int t, int r, int b) {
         }
     }
 
+    private static boolean isCvcMaximalLength(boolean isAmex, @Nullable String cvcText) {
```

Comment:
I can make a switch on that -- I was trying to encapsulate a smaller portion of the function this was torn out of, but I do end up just calling it anyway.